ios/engine/KMEI/KeymanEngine/Classes/Resource Management/ResourceFileManager.swift

```diff
@@ -137,7 +137,7 @@ public class ResourceFileManager {
    */
   public func prepareKMPInstall(from url: URL) throws -> KeymanPackage {
     // Once selected, start the standard install process.
-    log.info("Installing KMP from \(url)")
+    log.info("Opening KMP from \(url)")
```

Comment:
As it's now called as part of a lazy initializer for `Defaults.keyboardPackage`, "opening" is more accurate than "installing."